repositories/OptionRepository.js

```diff
@@ -27,7 +27,7 @@ const checkForExistingExclusive = async answerId => {
   }
 };
 
-const findAll = (where = {}, orderBy = "created_at", direction = "asc") =>
+const findAll = (where = {}, orderBy = "id", direction = "asc") =>
```

Comment:
Is ordering by `id` safe? Why doesn't order by `created_at` at work?  If the `created_at` becomes the same value when duplicating. Then maybe we should order by `created_at` and then fall back onto `id`?

Reply:
When we insert in during a duplication the `created_at` date is all the same so the order could not be determined. This resolved that issue. I don't see a point in having a fall back if `id` works just as well as `created_at`.

Reply:
Ordering by `id` should be safe enough until we have a requirement to re-order options.